src/main/java/com/klodnicki/Bike/v2/model/entity/User.java

```diff
@@ -29,71 +42,4 @@ public User(Long id, String name, String phoneNumber, String emailAddress, int a
         this.role = role;
         this.balance = balance;
     }
```

Comment:
This code is unused and if needed could be replaced with lombok annotation

Reply:
Why you don't want to have rent and bike in the constructor?

Reply:
I just don't need this :)

Reply:
What do you mean by not needing it? What is not needed here is this whole constructor, so please remove it as it is unused.

Reply:
OK! I have removed it and works fine!